docker-compose/README.md

```diff
@@ -13,84 +11,101 @@ These files provide example docker-compose files for setting up and configuring
+- `TAG`: The image tag, or version number, of the container images; defaults to `2.1.0`
+- `POP_ID`: Specifies the location (datacenter/pop) of the server where the data container is running; defaults to `mypop`
+- `SERVER_ID`: Identifies a specific server in a location where the data container is running; defaults to `myserver`
+- `DATA_PEERS`: Identifies the peer(s) of this host's data container with one operating as primary and the other replica; defaults to null
+- `DATA_HOSTS`: Series of comma delimited hostnames of data containers e.g. data1,data2; defaults to `data`
+- `DATA_CONTAINER_NAME`: Sets the container's name; defaults to `data`
+- `CORE_CONTAINER_NAME`: Sets the container's name; defaults to `core`
+- `XFR_CONTAINER_NAME`: Sets the container's name; defaults to `xfr`
+- `API_FQDN`: FQDN for the api and feed URLs; defaults to `api.mycompany.net`
+- `PORTAL_FQDN`: FQDN for the portal; defaults to `portal.mycompany.net`
+- `NAMESERVERS`: Nameservers used in SOA records; defaults to `ns1.mycompany.net`
+- `HOSTMASTER_EMAIL`: Hostmaster email address for SOA records; defaults to `hostmaster@mycompany.net`
+- `DNS_CONTAINER_NAME`: Sets the container's name; defaults to `dns`
+- `DHCP_CONTAINER_NAME`: Sets the container's name; defaults to `dhcp`
+- `DIST_CONTAINER_NAME`: Sets the container's name; defaults to `dist`
+- `CORE_HOSTS`: Series of comma delimited hostnames of `core` containers; specifies upstream `core` containers; defaults to `core`
+- `DIST_OR_CORE_HOSTS`: Series of comma delimited hostnames of distribution containers e.g. dist1,dist2; defaults to `dist`
+- `OPERATION_MODE`: `authoritative`/`recursive`; the mode of operation for this `dns` container; defaults to `authoritative`
 
 #### For example:
 
 ```shell
-$sudo TAG=1.1.0 docker-compose -p myproject -f docker-compose.yml up -d
+$sudo TAG=2.1.0 POP_ID=mypop SERVER_ID=myserver docker-compose -p myproject -f docker-compose.yml up -d
 ```
 
 ---
 
 
-# Resources for Core and Edge Facilities' Hosts
-Production and production-like deployment toplogies generally follow a "hub and spoke" pattern where certain services can be grouped in the "hub" referred to here as `core`; other services on the "spoke", which we referred to as `edge`.
+# Resources for Control and Edge Hosts
+Production and production-like deployment toplogies generally follow a "hub and spoke" pattern where certain services can be grouped in the "hub" referred to here as `control node`; other services on the "spoke", which we referred to as `edge node`.
 
-![Example topology with command line variables.](https://github.com/ns1/ns1-privatedns/blob/release/1.1.0/docker-compose/figure1.PNG?raw=true)
-**Figure 1**. Example topology with `core-compose.yml` and `edge-compose.yml` command line variables for reference.
+![Example topology with command line variables.](https://github.com/nsone/ns1-privatedns/blob/release/2.1.0-usage-info/docker-compose/figure2.PNG?raw=true)
```

Comment:
This link probably won't work as intended.  Image should be uploaded to this repo and link should point to that instead.